modules/ossm-vs-istio.adoc

```diff
@@ -126,7 +126,12 @@ You can deploy virtual machines to OpenShift using OpenShift Virtualization. The
 [id="ossm-gateways_{context}"]
 == Gateways
 
-{SMProductName} installs ingress and egress gateways by default. You can disable this in the SMCP using `spec.gateways.ingress.enabled=false` or `spec.gateways.egress.enabled=false`.
+{SMProductName} installs ingress and egress gateways by default. You can disable this in the `ServiceMeshControlPlane` (SMCP) resource by setting `spec.gateways.ingress.enabled=false` or `spec.gateways.egress.enabled=false`.
```

Comment:
Not sure if it's worth mentioning that they can be disabled together by `spec.gateways.enabled=false`.

Reply:
One setting instead of two?  Yep, worth adding that.